consensus/consortium/v2/snapshot.go

```diff
@@ -112,12 +113,48 @@ func loadSnapshot(
 	return snap, nil
 }
 
+// snapshot pruning
+// delete the nSnapshotsPrune oldest snapshots
+func (s *Snapshot) pruneSnapshot(db ethdb.Database, nSnapshotPrune int, chain consensus.ChainHeaderReader) error {
+	log.Info("Pruning snapshots at block", "block", s.Number, "nSnapshotPrune", nSnapshotPrune)
+	// Get block number to start pruning
+	curBlockNumber := s.Number
+	curBlockNumber -= curBlockNumber % 200 // make sure the block number is multiple of 200
+	curBlockNumber -= 200 * 144 * 5        // keep the last 5 days of snapshots
```

Comment:
We should create some constants for these expressions at the beginning of the file